mesonbuild/dependencies/ui.py

```diff
@@ -324,23 +324,10 @@ def _pkgconfig_detect(self, mods, kwargs):
             if prefix:
                 self.bindir = os.path.join(prefix, 'bin')
 
-    def _find_qmake(self, qmake):
-        # Even when cross-compiling, if a cross-info qmake is not specified, we
-        # fallback to using the qmake in PATH because that's what we used to do
-        if self.env.is_cross_build():
-            if 'qmake' in self.env.cross_info.config['binaries']:
-                return ExternalProgram.from_bin_list(self.env.cross_info.config['binaries'], 'qmake')
-        elif self.env.config_info:
-            # Prefer suffixed to unsuffixed version
-            p = ExternalProgram.from_bin_list(self.env.config_info.binaries, 'qmake-' + self.name)
-            if p.found():
-                return p
-            return ExternalProgram.from_bin_list(self.env.config_info.binaries, 'qmake')
-        return ExternalProgram(qmake, silent=True)
```

Comment:
Oh the rest is abstracted away, but I need to add that fallback back

Reply:
have you added the fallback?